it/nationaldutyrepaymentcenter/stubs/AmendCaseStubs.scala

```diff
@@ -18,15 +22,15 @@ trait AmendCaseStubs {
         |       "Description":"update request for Risk-2507: Value £199.99"
         |    }
         |}""".stripMargin,
-      """{
+      s"""{
         |    "Status": "Success",
         |    "StatusText": "Case Updated successfully",
-        |    "CaseID": "Risk-2507",
+        |    "CaseID": "$caseRef",
```

Comment:
OBSERVATION: If we have the caseRef as a variable, should the CaseID (7 lines up) not also use that variable?  This is a stub in a test, so tbh not something I am expecting to change.